### PR TITLE
Config with pure python template f3dcda6c

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,6 +4,8 @@ Changes
 1.0 (unreleased)
 ----------------
 
+* Add support for Python 3.9, 3.10, 3.11.
+
 * Drop support for Python 2.7, 3.5, 3.6.
```
Comment (mauritsvanrees) [inline CHANGES.rst:9]: These lines should be below "Breaking changes." Or remove those sub headers if you want.

**Review by mauritsvanrees — APPROVED** [review]
LGTM. Very minor thing to fix in the changelog.

BTW, Plone 6 does not use this, we have `plone.autoinclude`.

File: CHANGES.rst (hunk 1 above)
Comment (icemac) [reply CHANGES.rst]: Fixed in https://github.com/zopefoundation/z3c.autoinclude/pull/16/commits/e4ef126645177f361eb976934eef9c3b6bd7ae26

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**